scripts/coverage_diff.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python3
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+
+# Install Intel FPGA SDK for OpenCL Pro Edition
+# https://fpgasoftware.intel.com/
+
+# Example usage: python scripts/coverage_diff.py child_build_dir parent_build_dir
+import xml.etree.ElementTree as ET
+import sys
+import argparse
+import glob
+
+parser = argparse.ArgumentParser(description='Optional app description')
+parser.add_argument('child_coverage', type=str,
+                    help='Child branch build directory')
+parser.add_argument('parent_coverage', type=str,
+                    help='Parent branch build directory')
+args = parser.parse_args()
```

Comment:
You can move the argument parsing and other top-level code into a `main` function, which ensures the top-level variables are scoped to that function and cannot be used from other functions, which in turn ensures easier-to-comprehend functional code without global variables. Sometimes this is not necessary, but since you already have functions (also see duplication comment below) it does not hurt to go all the way with a `main` function.